### ISSUE TITLE
InflateException on inflating class com.goodayapps.widget.AvatarView

### ISSUE THREAD

**Issue by romasworld** [helpcrunch/android-sdk-demo#80]
**Environment**
**Help Crunch Version**: 3.3.6
**Android Version**: 12
**Affected Devices:** Pixel 4 XL, Pixel 6 Pro
**Non-Affected Devices:** Samsung S22 (indicating the issue might be device-specific)

The crash occurs immediately, making the chat feature unusable on the affected devices.

**Stack Trace**
```
android.view.InflateException: Binary XML file line #15 in layout/layout_hc_agents_item: Binary XML file line #15 in layout/layout_hc_agents_item: Error inflating class com.goodayapps.widget.AvatarView
Caused by: android.view.InflateException: Binary XML file line #15 in layout/layout_hc_agents_item: Error inflating class com.goodayapps.widget.AvatarView
Caused by: java.lang.reflect.InvocationTargetException
at java.lang.reflect.Constructor.newInstance0(Native Method)
at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
at android.view.LayoutInflater.createView(LayoutInflater.java:858)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1010)
at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:965)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:1127)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1088)
at android.view.LayoutInflater.inflate(LayoutInflater.java:686)
at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
at android.view.LayoutInflater.inflate(LayoutInflater.java:485)
at android.view.View.inflate(View.java:27713)
at com.helpcrunch.library.utils.views.toolbar.HCAgentsView.a(SourceFile:345)
at com.helpcrunch.library.utils.views.toolbar.HCAgentsView.a(SourceFile:344)
at com.helpcrunch.library.utils.views.toolbar.HCAgentsView.a(SourceFile:337)
at com.helpcrunch.library.utils.views.toolbar.HCAgentsView.b(SourceFile:15)
at com.helpcrunch.library.utils.views.toolbar.HCAgentsView.a(SourceFile:19)
at com.helpcrunch.library.utils.views.toolbar.HCToolbarView.a(SourceFile:439)
at com.helpcrunch.library.ui.screens.chat.HcChatFragment.a(SourceFile:3964)
at com.helpcrunch.library.ui.screens.chat.HcChatFragment.a(SourceFile:5)
at com.helpcrunch.library.ui.screens.chat.HcChatFragment$onBindLiveData$8.a(SourceFile:1)
at com.helpcrunch.library.ui.screens.chat.HcChatFragment$onBindLiveData$8.invoke(SourceFile:1)
at com.helpcrunch.library.ui.screens.chat.HcChatFragment$sam$androidx_lifecycle_Observer$0.onChanged(Unknown Source:2)
at androidx.lifecycle.LiveData.considerNotify(LiveData.java:133)
at androidx.lifecycle.LiveData.dispatchingValue(LiveData.java:146)
at androidx.lifecycle.LiveData$ObserverWrapper.activeStateChanged(LiveData.java:483)
at androidx.lifecycle.LiveData$LifecycleBoundObserver.onStateChanged(LiveData.java:440)
at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.kt:322)
at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.kt:258)
at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.kt:294)
at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.kt:143)
at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.kt:126)
at androidx.fragment.app.FragmentViewLifecycleOwner.handleLifecycleEvent(FragmentViewLifecycleOwner.java:100)
at androidx.fragment.app.Fragment.performStart(Fragment.java:3194)
at androidx.fragment.app.FragmentStateManager.start(FragmentStateManager.java:628)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:290)
at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
at androidx.fragment.app.FragmentManager.dispatchStart(FragmentManager.java:2959)
at androidx.fragment.app.FragmentController.dispatchStart(FragmentController.java:274)
at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:358)
at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:251)
at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1510)
at android.app.Activity.performStart(Activity.java:8315)
at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3701)
at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2308)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loopOnce(Looper.java:201)
at android.os.Looper.loop(Looper.java:288)
at android.app.ActivityThread.main(ActivityThread.java:7898)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Caused by: android.content.res.Resources$NotFoundException: Font resource ID #0x7f090009 could not be retrieved.
at androidx.core.content.res.ResourcesCompat.loadFont(ResourcesCompat.java:570)
at androidx.core.content.res.ResourcesCompat.getFont(ResourcesCompat.java:411)
at com.goodayapps.widget.utils.TypedArrayKt.getTypefaceOrNull(TypedArray.kt:25)
at com.goodayapps.widget.AvatarView.configureStyleValues(AvatarView.kt:239)
at com.goodayapps.widget.AvatarView.init(AvatarView.kt:152)
at com.goodayapps.widget.AvatarView.<init>(AvatarView.kt:123)
... 57 more
```

**Comment by TalbotGooday** [comment]
Hi! Thanks for reporting! Could you please share your org name here or via email: alexey.mostovoy.w@gmail.com

**Comment by kolega69** [comment]
Hi! Same occured to us

Model: Pixel 6 Pro 
Android Version: 12
Help Crunch Version: 3.3.7

**Comment by TalbotGooday** [comment]
Fixed in 3.3.8
